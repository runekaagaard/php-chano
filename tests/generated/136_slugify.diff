001- jack-jill-like-numbers-123-and-4-and-silly-characters
001+ Fatal error: Call to undefined method DtlIter::slugify() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/django-generated/136_slugify.php on line 4