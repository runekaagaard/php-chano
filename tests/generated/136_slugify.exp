jack-jill-like-numbers-123-and-4-and-silly-characters