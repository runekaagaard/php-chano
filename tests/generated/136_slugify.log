
---- EXPECTED OUTPUT
jack-jill-like-numbers-123-and-4-and-silly-characters
---- ACTUAL OUTPUT
Fatal error: Call to undefined method DtlIter::slugify() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/generated/136_slugify.php on line 4
---- FAILED
