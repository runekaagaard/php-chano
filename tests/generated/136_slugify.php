<?php
include dirname(__FILE__) . '/../../DtlIter.php';
$items = new DtlIter(array(array('input' => ' Jack & Jill like numbers 1,2,3 and 4 and silly characters ?%.$!/')));
foreach ($items as $i) echo $i->input->slugify();
