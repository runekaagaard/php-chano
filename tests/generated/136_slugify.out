Fatal error: Call to undefined method DtlIter::slugify() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/generated/136_slugify.php on line 4